src/community/geogig/pom.xml

```diff
@@ -24,11 +24,11 @@
   </properties>
 
   <repositories>
-    <repository>
-      <id>repo.locationtech.org</id>
-      <name>GeoGig Repository</name>
-      <url>https://repo.locationtech.org/content/repositories/geogig/</url>
-    </repository>
+<!--    <repository>-->
+<!--      <id>repo.locationtech.org</id>-->
+<!--      <name>GeoGig Repository</name>-->
+<!--      <url>https://repo.locationtech.org/content/repositories/geogig/</url>-->
+<!--    </repository>-->
```

Comment:
This repo has been offline for a while, and I don't believe it is coming back anytime soon, so I think you can just delete this section rather than comment it out.